sound/soc/sof/ipc4-loader.c

```diff
@@ -112,6 +112,7 @@ static ssize_t sof_ipc4_fw_parse_ext_man(struct snd_sof_dev *sdev,
 				return -EINVAL;
 			}
 
+			fw_module->fm_config = &fm_config[fm_entry->cfg_offset];
```

Comment:
so for all cases of saving a pointer to reuse later, you'll see me ask:

How safe if this, and is the pointer valid always?

Reply:
@plbossart, yes, I have asked this and since we are keeping the firmware loaded all the time, this is OK.

Reply:
ok